config/config.exs

```diff
@@ -2,6 +2,7 @@ import Config
 
 config :ash, :validate_domain_resource_inclusion?, false
 config :ash, :validate_domain_config_inclusion?, false
+config :ash, :custom_expressions, [AshAdmin.Expressions.Position]
```

Comment:
We won't be able to use a custom expression like this unfortunately, because once this is deployed the config would be the *target* users config. We'll have to inline it.

Reply:
Ah, gotcha. I'll take another stab at inlining it.